inspec/a2-iam-v2-integration/controls/iam_v2.rb

```diff
@@ -332,19 +332,20 @@
   end
 
   describe "v2beta project API" do
+    custom_project_id = "inspec-custom-project-api-test-#{TIMESTAMP}"
```

Comment:
why isn't this with the constants above?

Reply:
Each describe needs to have a different ID because now project delete isn't synchronous :/